src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -273,32 +262,14 @@ class HaConfigHardware extends SubscribeMixin(LitElement) {
         .header=${this.hass.localize("ui.panel.config.hardware.caption")}
       >
         ${isComponentLoaded(this.hass, "hassio")
-          ? html`<ha-button-menu corner="BOTTOM_START" slot="toolbar-icon">
+          ? html`
               <ha-icon-button
+                slot="toolbar-icon"
                 .label=${this.hass.localize("ui.common.menu")}
                 .path=${mdiDotsVertical}
-                slot="trigger"
+                @click=${this._openRestartMoveDialog}
               ></ha-icon-button>
-              <mwc-list-item @click=${this._openHardware}
-                >${this.hass.localize(
-                  "ui.panel.config.hardware.available_hardware.title"
-                )}</mwc-list-item
-              >
-              ${this._hostData
```

Comment:
@bramkragten Why do we this condition. Isn't `isComponentLoaded(this.hass, "hassio")` enough ?

Reply:
The reason is that we can not reboot/shut down supervised systems (My comment earlier was missing that part).

Reply:
Thank you, I will add this condition too.

Reply:
In case of HA container - will the dialog have a possibility to reboot a host?
Also, the 2nd and 3rd line should probably be “merged” into one in this case.

Reply:
The code handle this case